dev_guide/managing_images.adoc

```diff
@@ -1099,6 +1099,65 @@ my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
 ----
 ====
 
+
+[[triggering-updates-on-image-stream-changes]]
+== Triggering Updates On Image Stream Changes
+
+=== Overview
```

Comment:
I would recommend taking out the '=== Overview'

Reply:
done